scripts/build-rir.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -xe
+
+SRC_DIR=${SRC_DIR:-$PWD}
+BUILD_DIR=${BUILD_DIR:-$PWD/rkt-build}
+RKT_BUILDER_ACI=${RKT_BUILDER_ACI:-coreos.com/rkt/builder:v0.10.0}
+
+mkdir -p $BUILD_DIR
+
+rkt run \
+    --volume src-dir,kind=host,source=$SRC_DIR \
+    --volume build-dir,kind=host,source=$BUILD_DIR \
+    --interactive \
+    --insecure-skip-verify \
+    $RKT_BUILDER_ACI \
```

Comment:
Here, `$RKT_BUILDER_ACI` can be either a local file or the aci from `coreos.com`.

`--insecure-skip-verify` should only be use if we work on a local file since we don't want to disable gpg signature checks when downloading from internet.